controllers/managedfusionoffering_controller.go

```diff
@@ -133,7 +135,7 @@ func (r *ManagedFusionOfferingReconciler) reconcileOperatorGroup() error {
 		if err := r.own(r.operatorGroup); err != nil {
 			return err
 		}
-		r.operatorGroup.Spec = pluginGetDesiredOperatorGroupSpec(r)
+		r.operatorGroup.Spec = pluginGetDesiredOperatorGroupSpec(r.managedFusionOffering)
```

Comment:
Not sure why you decided to stop sending the reconciler, please add it back
```suggestion
		r.operatorGroup.Spec = pluginGetDesiredOperatorGroupSpec(r, r.managedFusionOffering)
```